src/main/java/programminglife/ProgrammingLife.java

```diff
@@ -31,5 +39,38 @@ public static void main(String[] args) {
         } catch (FileNotFoundException e) {
             System.err.println(String.format("File not found (%s)", graphFile));
         }
+        launch(args);
```

Comment:
Please do not launch this in main, because this makes manual more difficult and does not yet have functionality.